config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```diff
@@ -1220,3 +1920,211 @@ fields:
             default:
               region: content
               weight: 100
+
+    -   name: bchem_labfacility
+        content_type: biochem_experiment
+        label: Laboratory Facility
+        type: chado_property_type_default
+        description: A research facility in which systems of manufactured products control internal conditions and in which scientific or technological research, experiments, and measurement may be performed.
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: ENVO
+            termAccession: '01001406'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_date
+        content_type: biochem_experiment
+        label: Experiment Date
+        type: chado_property_type_default
+        description: The date and time the experiment was performed (e.g. 2024Aug5 3:15pm)
+        cardinality: -1
+        required: false
+        storage_settings:
+            storage_plugin_id: chado_storage
+            storage_plugin_settings:
+                base_table: project
+                prop_table: projectprop
+        settings:
+            termIdSpace: NCIT
+            termAccession: 'C29848'
+        display:
+          view:
+            default:
+              region: content
+              label: above
+              weight: 20
+          form:
+            default:
+              region: content
+              weight: 20
+
+    -   name: bchem_design
+        content_type: biochem_experiment
+        label: Experimental Design
+        type: chado_property_type_default
+        description: Describe the design of your experiment in detail. More specifically, include information on replication, randomization and blocking. Also include the location-year combinations you intend to collect data in. Make sure these match those specifieid in "Experiment Locations" and "Experiment Years".
```

Comment:
This help text does not make sense for a Biochemical assay. I'm actually not sure biochemical assays have an equivalent to Experimental design in field experiments 🤔 Unless we describe this as plate layouts, etc?

Reply:
Right 🙈 Experimental Design shouldn't be here at all. 
If we don't have an equivalent to Experimental design for biochemical assays, is there any universal design-wise procedure that can be included here?

Reply:
Might it already be included in Experiment Summary? Maybe we should add an assay protocol type field and/or biochemical technique (i.e. "Optical Coherence Tomography").